drivers/spi/Kconfig

```diff
@@ -54,11 +54,16 @@ config SPI_STM32
 	bool
 	prompt "STM32 MCU SPI controller driver"
 	depends on SPI && SOC_FAMILY_STM32
-	depends on SOC_SERIES_STM32L4X || SOC_SERIES_STM32F4X || SOC_SERIES_STM32F3X
+	depends on SOC_SERIES_STM32L4X || SOC_SERIES_STM32F0X || SOC_SERIES_STM32F3X || SOC_SERIES_STM32F4X
 	default n
 	help
 	  Enable SPI support on the STM32 family of processors.
 
+config SPI_STM32_HAS_FIFO
+	bool
+	depends on SPI_STM32
+	default y if SOC_SERIES_STM32L4X || SOC_SERIES_STM32F0X || SOC_SERIES_STM32F3X
```

Comment:
Just curious:
Why not adding a second "depends on" line to specify the SERIES?

Reply:
Yes, I should